CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 <!-- ### Added -->
 <!-- ### Removed -->
 
+### Added
+* Added `enableUpdating()` to `UpdatingElement` to enable customizing when updating is enabled.
```

Comment:
SInce it doesn't look like we have issues for these, can you link to this PR?

Reply:
Done